JS-Algorithms/challenges.js

```diff
@@ -70,7 +87,15 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  // YOUR CODE HERE...
+  let sum = 0n;
```

Comment:
Is this the same?

```suggestion
  let sum = BigInt(0);
```

Reply:
Yes they are both BigInt value with different declaration. BigInt() makes it more readable and 'n' makes it a bit more simple but less readable.